jekyll/_layouts/classic-docs.html

```diff
@@ -1,58 +1,138 @@
 <!DOCTYPE html>
 <html lang="{{ site.lang }}">
-{% include_cached head.html site=site page=page %}
+{% include head.html %}
 <body class="{% if page.page-type == "index" or page.page-type == "homepage" %}homepage{% endif %}">
 {% if jekyll.environment == "production" %}
-  {% include_cached google-tag-manager.html %}
+	{% include google-tag-manager.html %}
 {% endif %}
 
 <div class="outer">
-  {% include_cached global-nav.html site=site page=page %}
-  {% include_cached hits-target.html %}
-  <div class="main-body">
-    {% assign lang = page.lang %}
-    {% assign sidenav = site.data.sidenav[lang] %}
-    {% assign current_url = page.url | slice: 1, page.url.size %}
-    {% include_cached sidebar.html lang=lang sidenav=sidenav current_url=current_url%}
-    <div class="article-container docs" id="main">
-      {% if page.collection == "ccie" %}{% include_cached v2-banner.html %}{% endif %}
-      {% if page.collection == "cci1" %}{% include_cached v2-dev-banner.html %}{% endif %}
-      {% if page.page-type == "index" %}<h2>{{ page.title }}</h2>{% endif %}
-
-
-      <article>
-        <!-- Title and version badges: -->
-        <div>
-          {% if page.page-type != "index" and page.page-type != "homepage" %}<h1 id="{{ page.title | downcase | split: " " | join: "-" }}">{{ page.title }}</h1>{% endif %}
-        </div>
-        {% if page.version %}
-        <div class="server-version-wrapper">
-          <div id="tooltip" role="tooltip" class="server-version-popover">
-            This document is applicable to the following:
-            <div id="arrow" data-popper-arrow></div>
+	{% include_localized global-nav.html %}
+	{% include main-searchbar.html %}
+	{% include mobile-sidebar.html %}
```

Comment:
`main-searchbar` and `mobile-sidebar` were both removed earlier in master, and it seems like you've added them back here when they should probably stay removed.